scvi/harmonization/DEfunctions.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import pandas as pd
+
+use_cuda = True
+
+
+def GetScale(trainer, n_batches, n_samples_per_batch, gene_dataset, selection, seed=0):
+    scale = []
+    np.random.seed(seed)
+    for batch in range(n_batches):
```

Comment:
Some of these functions should be encapsultated in the posterior object themselves and not as a new module